test/shared/src/main/scala/zio/test/TestClock.scala

```diff
@@ -87,10 +87,9 @@ import scala.collection.immutable.SortedSet
 trait TestClock extends Clock with Restorable {
   def adjust(duration: Duration)(implicit trace: Trace): UIO[Unit]
   def adjustWith[R, E, A](duration: Duration)(zio: ZIO[R, E, A])(implicit trace: Trace): ZIO[R, E, A]
-  def setDateTime(dateTime: OffsetDateTime)(implicit trace: Trace): UIO[Unit]
-  def setTime(duration: Duration)(implicit trace: Trace): UIO[Unit]
+  def setInstant(instant: Instant)(implicit trace: Trace): UIO[Unit]
```

Comment:
Should we just rename this to `setTime` now? Isn't the overall point of this that we want to be able to set the time but doing so as a duration since the epoch isn't the best way to do it?

Reply:
done